setup/pi/create-backingfiles.sh

```diff
@@ -145,10 +162,27 @@ rm -f "$CAM_DISK_FILE_NAME"
 rm -f "$MUSIC_DISK_FILE_NAME"
 rm -rf "$BACKINGFILES_MOUNTPOINT/snapshots"
 
+if [ "$USE_EXFAT" = true  ]
+then
+  # Check if kernel supports ExFAT 
+  if ! cat /proc/filesystems | grep -q exfat &> /dev/null
```

Comment:
Not sure if you saw the shellcheck failure or if those are only visible to me, but this fails shellcheck. Use `grep -q exfat /proc/filesystems` instead.

Reply:
Done. I had not installed shellcheck. Thanks.